backend/src/test/java/gov/cdc/usds/simplereport/validators/FileValidatorTest.java

```diff
@@ -0,0 +1,304 @@
+            "group home is not an acceptable value for column resident_congregate_setting",
+            "n/a is not an acceptable value for column employed_in_healthcare",
+            "doctor is not an acceptable value for column role",
+            "Alaska is not an acceptable value for column state",
+            "1234 is not a valid value for column zip_code",
+            "America is not an acceptable value for column country",
+            "4108675309 is not a valid value for column phone_number",
+            "cell is not an acceptable value for column phone_number_type",
+            "janedoe.com is not a valid value for column email");
+  }
+
+  @Test
+  void invalidValuesAndMissingFieldsInMultipleRows_returnError() {
+    // GIVEN
+    InputStream input =
+        loadCsv("patientBulkUpload/invalidValuesAndMissingFieldsInMultipleRows.csv");
+    // WHEN
+    List<FeedbackMessage> errors = patientBulkUploadFileValidator.validate(input);
+    // THEN
+    assertThat(errors).hasSize(3);
+    assertThat(errors.get(0).getScope()).isEqualTo("item");
+    assertThat(errors.get(0).getMessage())
+        .isEqualTo("african american is not an acceptable value for column race");
+    assertThat(errors.get(0).getIndices()).isEqualTo(List.of(2, 3));
+    assertThat(errors.get(1).getScope()).isEqualTo("report");
+    assertThat(errors.get(1).getMessage()).isEqualTo("ethnicity is a required column.");
+    assertThat(errors.get(1).getIndices()).isEqualTo(List.of(4, 5));
+    assertThat(errors.get(2).getScope()).isEqualTo("report");
+    assertThat(errors.get(2).getMessage()).isEqualTo("race is a required column.");
+    assertThat(errors.get(2).getIndices()).isEqualTo(List.of(4, 5));
+  }
+
+  @Test
+  void emptyRow_returnsError() {
+    // GIVEN
+    InputStream input = loadCsv("patientBulkUpload/emptyRow.csv");
+    // WHEN
+    List<FeedbackMessage> errors = patientBulkUploadFileValidator.validate(input);
+    // THEN
+    assertThat(errors).hasSize(12);
+    List<String> errorMessages =
+        errors.stream().map(FeedbackMessage::getMessage).collect(Collectors.toList());
+    assertThat(errorMessages)
+        .contains(
+            "first_name is a required column.",
+            "last_name is a required column.",
+            "race is a required column.",
+            "date_of_birth is a required column.",
+            "biological_sex is a required column.",
+            "ethnicity is a required column.",
+            "street is a required column.",
+            "state is a required column.",
+            "zip_code is a required column.",
+            "phone_number is a required column.",
+            "employed_in_healthcare is a required column.",
+            "resident_congregate_setting is a required column.");
+  }
+
+  private InputStream loadCsv(String csvFile) {
+    return FileValidatorTest.class.getClassLoader().getResourceAsStream(csvFile);
+  }
+
+  @Test
+  void testValidFile() {
```

Comment:
nit: can you specify in the test name that this is for results instead of patients?